src/Form/MetadataDisplayForm.php

```diff
@@ -109,7 +109,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
     $form['preview']['render_native'] = [
       '#type' => 'checkbox',
       '#defaut_value' => FALSE,
-      '#title' => 'Show Preview using native Output Format (e.g HTML)',
+      '#title' => 'Show Preview using native Output Format (e.g HTML) and output errors if there are any.',
```

Comment:
Can you maybe move this to a description?
If errors are found Preview will fail. To checkboxes show "#descrition?"

Reply:
@DiegoPino Do you mean have the description only show if the option is checked? If so I couldn't figure it how to do that in time, but I moved it to the description in any case. It seems like it needs to be updated in `ami.module`? Not exactly sure why yet, but the element there seems to override this one.

Reply:
@aksm what I mean is to use the #description key in the render array